### PR TITLE
Contribution and Conduct guidelines

### PR DESCRIPTION
**Pull request by benkoller**
This pull request includes a first draft for this project's Contributor License Agreement, Code of Conduct as well as Contribution guidelines.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+2. Clone the project to your machine
+3. Create a branch locally with a succinct but descriptive name
+4. Commit changes to the branch
+5. Following any formatting and testing guidelines specific to this repo
+6. Push changes to your fork
+7. Open a PR in our repository and follow the PR template so that we can efficiently review the changes.
+
+## Getting Help
+
+Join us in the [ZenML Slack Community](https://join.slack.com/t/zenml/shared_invite/zt-ji4womyb-ONoRCMjJEBF_YjiAV30YVg) and post your question there.
```
Comment (hamzamaiot) [inline CONTRIBUTING.md:57]: Will this link expire?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Contributing to Auth0 projects
```
Comment (hamzamaiot) [inline CONTRIBUTING.md:1]: @benkoller Do we need to write Auth0 projects here?

File: CLA.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
+# Fiduciary License Agreement 2.0
+based on the
+## Individual Contributor Exclusive License Agreement
+(including the Traditional Patent License OPTION)
+
+Thank you for your interest in contributing to maiot GmbH's ZenML ("We" or "Us").
+
+The purpose of this contributor agreement ("Agreement") is to clarify and document the rights granted by contributors to Us. To make this document effective, please follow the instructions at https://zenml.io/cla.
```
Comment (hamzamaiot) [inline CLA.md:8]: @benkoller  https://zenml.io/cla is broken for me, with:
```
<Error>
<Code>NoSuchKey</Code>
<Message>The specified key does not exist.</Message>
<Key>cla</Key>
<RequestId>BC64289244063D73</RequestId>
<HostId>bQcQ/5mua5KyAK3EUCbSPZfwk7HrKzhGKTkL6P7tnFAwO8NpOAqXuhzxax+EZiBLaiR7XQczav0=</HostId>
</Error>
```

**Review by hamzamaiot — COMMENTED** [review]
Looks great! Just some small comments on minor things. Kindly do take a look and update if necessary

File: CONTRIBUTING.md (hunk 2 above)
Comment (benkoller) [reply CONTRIBUTING.md:1]: Nope. I did not proofread thoroughly enough. Will resolve with the next commit.

File: CLA.md (hunk 3 above)
Comment (benkoller) [reply CLA.md:8]: Just missing a backslash. Added it, will resolve with the next commit.

File: CONTRIBUTING.md (hunk 1 above)
Comment (benkoller) [reply CONTRIBUTING.md:57]: Semi-resolved via static redirect on zenml.io/slack-invite/

**Comment by abhiksark** [comment]
In slack it is asking email with @maiot.io address. Is slack open for general public?

**Comment by hamzamaiot** [comment]
> @benkoller In slack it is asking email with @maiot.io address. Is slack open for general public?

@abhiksark Thanks for bringing that to our attention! The slack link should work now and be open to public! Its https://zenml.io/slack-invite. Please feel free to join!

**Pull request merged**